diff --git a/node_modules/expo-modules-core/src/index.ts b/node_modules/expo-modules-core/src/index.ts
index 3be4bbf..c8a16f4 100644
--- a/node_modules/expo-modules-core/src/index.ts
+++ b/node_modules/expo-modules-core/src/index.ts
@@ -1,7 +1,8 @@
 import './sweet/setUpErrorManager.fx';
 import './web';
 
-export type * from './ts-declarations/global';
+// SWC can't parse `export type *` yet – switch to a normal re-export
+export * from './ts-declarations/global';
 
 export { default as NativeModule } from './NativeModule';
 export { default as SharedObject } from './SharedObject';
